src/export_members.ts

```diff
@@ -42,6 +42,7 @@ async function getAllMembers(
       root_account: p.root_account,
       controller_account: p.controller_account,
       handle: p.handle,
+      name: ``,
```

Comment:
we modified the expected json format in the node chainspec so adding this field is no longer necessary.